### ISSUE TITLE
Non Quay UI flow

### ISSUE THREAD

**Issue by vrabbi** [konflux-ci/konflux-ci#317]
It would be great if the UI flow for onboarding supported not only quay but also common registries like Harbor, ECR, ACR, GCR, Artifactory, Nexus etc. 
I personaly use Harbor which is also the easiest to integrate with as no pre creation of a repo is needed. 
Something similar to the jx-scm (go-scm wrapper) tool from JenkinsX but for OCI registries seems like the right approach in my mind by building a simple abstraction layer above the different registries for common functionality needed by Konflux.

**Comment by gbenhaim** [comment]
Thank you for opening this issue @vrabbi. I agree 100% that more registries should be supported.
We work on removing `image-controller` (the service that provisions repositories on quay) from konflux-core, and make it possible for users to specify in the UI the repository they would like to use (in any registry).

We still didn't define the a generic interface for automatic provisioning of repositories. The way I see it, I would love us to have a plug able architecture were additional provisioners can be added to Konflux independently from each other.

**Comment by mrusso99** [comment]
Hi @gbenhaim any updates on this?

**Comment by gbenhaim** [comment]
I don't think it was implemented in konflux-ci/konflux-ui already.
@sahil143 please keep me honest here.

**Comment by sahil143** [comment]
That's correct. It isn't fully implemented in konflux-ui yet, as some paths still depend on the Image Controller. I've opened an issue to track this https://github.com/konflux-ci/konflux-ui/issues/1138
